entities/jackson/pom.xml

```diff
@@ -23,5 +23,12 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
```

Comment:
is it possible to tell the class generator to use guava for the equals, hashCode, and toString methods? we already have guava included as a dependency and I wouldn't want to force another dependency on clients using this module if possible.